### PR TITLE
Add io.github.hermitdemschoenenleben.linien

### PR DESCRIPTION
**Pull request by hermitdemschoenenleben**
Linien is a scientific lab software that allows for stabilizing the frequency of lasers.

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build io.github.hermitdemschoenenleben.linien

File: io.github.hermitdemschoenenleben.linien.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+app-id: io.github.hermitdemschoenenleben.linien
+runtime: org.kde.Platform
+runtime-version: '5.12'
```
Comment (LeandroStanger) [inline io.github.hermitdemschoenenleben.linien.yml:3]: Update runtime to 5.14
```suggestion
runtime-version: '5.14'
```

File: io.github.hermitdemschoenenleben.linien.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          - processed=`sed -e 's|prefix|sysroot|' <<< $@`
+          - python3 configure.py $processed
+        dest-filename: configure
+    cleanup:
+      - /share/sip
+
+  - python3-cython.json
+  - python3-wheel.json
+  - python3-numpy.json
+  - lapack.json
```
Comment (barthalion) [inline io.github.hermitdemschoenenleben.linien.yml:81]: Any reason to keep lapack/boost in separate files? Besides, you can generate modules for python dependencies with https://github.com/flatpak/flatpak-builder-tools/tree/master/pip

File: io.github.hermitdemschoenenleben.linien.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+app-id: io.github.hermitdemschoenenleben.linien
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: launcher.sh
+finish-args:
+  - --socket=x11
```
Comment (barthalion) [inline io.github.hermitdemschoenenleben.linien.yml:7]: You want `--device=dri` for GPU acceleration. Not crucial but will improve GUI performance.

File: io.github.hermitdemschoenenleben.linien.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline io.github.hermitdemschoenenleben.linien.metainfo.xml:1]:
```
builddir/files/share/appdata/io.github.hermitdemschoenenleben.linien.appdata.xml: FAILED:
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <description> required
```

**Comment by hermitdemschoenenleben** [comment]
In the latest commit I
 - updated runtime to 5.14
 - added missing stuff to metainfo.xml
 - added --device=dri
 - combined lapack+scipy into a single file

File: io.github.hermitdemschoenenleben.linien.yml (hunk 2 above)
Comment (hermitdemschoenenleben) [reply io.github.hermitdemschoenenleben.linien.yml]: no, there was no reason for this, I just changed it

File: io.github.hermitdemschoenenleben.linien.yml (hunk 3 above)
Comment (hermitdemschoenenleben) [reply io.github.hermitdemschoenenleben.linien.yml:7]: ah, great! Thanks, I just added this option

**Comment by bilelmoussaoui** [comment]
bot, build io.github.hermitdemschoenenleben.linien

**Comment by hermitdemschoenenleben** [comment]
When trying the test build on different computers, I noticed some problems that I think are related to --device=dri. I just disabled dri in the last commit. Could you trigger a new build, please?

**Comment by barthalion** [comment]
bot, build io.github.hermitdemschoenenleben.linien

**Comment by hermitdemschoenenleben** [comment]
great, this build seems to work!

**Pull request closed without merging**